databrowser/src/domain/datasets/ui/toolBox/ToolBox.vue

```diff
@@ -5,167 +5,109 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
 <template>
-  <!-- Mobile toolBox open button -->
-  <ButtonCustom
-    v-if="!toolBoxStore.visible"
-    class="absolute bottom-10 right-0 z-20 flex rotate-0 items-center justify-center rounded-r-none shadow-[0_10px_15px_-3px_rgba(0,0,0,0.4)] sm:block md:hidden"
-    data-test="mobile-open-toolBox"
-    @click="toolBoxStore.toggleToolboxVisibility(!toolBoxStore.visible)"
-  >
-    <IconAdd class="mr-2 size-5" />
-    <span>Toolbox</span>
-  </ButtonCustom>
 
   <!-- ToolBox content -->
   <div
-    class="absolute top-0 z-10 flex h-full flex-col overflow-x-auto bg-gray-50 transition-all md:relative"
+    class="absolute top-0 z-[10] flex h-full flex-col overflow-x-auto bg-gray-50 transition-all md:relative"
     :class="{
-      'w-full md:w-1/3': toolBoxStore.visible,
-      'w-0 md:w-[4.5rem]': !toolBoxStore.visible,
+      'w-full md:w-1/3': activeSection,
+      'w-0': !activeSection,
       'bg-white': isWhite,
     }"
   >
     <div
-      class="flex flex-1 overflow-y-auto md:block md:pl-14"
-      :class="{ block: toolBoxStore.visible, hidden: !toolBoxStore.visible }"
+      class="flex flex-1 overflow-y-auto md:block"
+      :class="{ block: activeSection, hidden: !activeSection }"
     >
       <Transition
         @after-enter="
           (el: Element) => (el.classList.value = 'block h-full w-full')
         "
         @before-leave="(el: Element) => (el.classList.value = 'hidden')"
       >
-        <div v-if="toolBoxStore.visible" :class="{ hidden: !mdAndLarger }">
-          <ContentAlignmentX class="h-full">
+        <div v-if="activeSection" :class="{ hidden: !mdAndLarger }">
+          <ContentAlignmentX class="h-full" :noPadding="true" >
```

Comment:
I'd rather use a `div` here or create a dedicated `ToolboxAlignmentX` component (if reused several times), instead of modifying the `ContentAlignmentX` class just for this special purpose. Having `ContentAlignmentX` reused here makes the code harder to understand. My thoughts where: "why does `ContentAlignmentX` now support a `noPadding` prop? => ah, it's because it is reused once somewhere else where it is necessary to have no padding..."

Not reusing `ContentAlignmentX` would simplify the code